nncf/openvino/pot/qunatization/quantize.py

```diff
@@ -0,0 +1,476 @@
+    """
+    with tempfile.TemporaryDirectory(dir=tempfile.gettempdir()) as tmp_dir:
+        xml_path = str(Path(tmp_dir) / 'model.xml')
+        bin_path = str(Path(tmp_dir) / 'model.bin')
+        ov.serialize(model, xml_path, bin_path)
+        model_config = {
+            'model_name': 'model',
+            'model': xml_path,
+            'weights': bin_path,
+        }
+        pot_model = pot.load_model(model_config, target_device)
+
+    return pot_model
+
+
+def _convert_compressed_model_to_openvino_model(model: pot.graph.nx_model.CompressedModel) -> ov.Model:
+    """
+    Saves the provided POT compressed model and loads it as `openvino.runtime.Model` object.
+
+    :param model: The POT compressed model.
+    :return: The `openvino.runtime.Model`  object which represents the provided model.
+    """
+    with tempfile.TemporaryDirectory(dir=tempfile.gettempdir()) as tmp_dir:
+        paths = pot.save_model(model, save_path=tmp_dir, model_name='model')
+        xml_path = paths[0]['model']
+        bin_path = paths[0]['weights']
+        ie = ov.Core()
+        ov_model = ie.read_model(xml_path, bin_path)
+    return ov_model
+
+
+def _create_ignored_scope_config(ignored_scope: Optional[IgnoredScope]) -> Dict[str, Any]:
+    """
+    Maps the content of `IgnoredScope` class to the `ignored` section of POT config.
+
+    :param ignored_scope: The ignored scope
+    :return: A POT ignored scope configuration as dict
+    """
+    if ignored_scope is None:
+        return {}
+
+    ignored = {}
+    if ignored_scope.names is not None:
+        ignored['scope'] = ignored_scope.names
+    if ignored_scope.patterns:
+        raise RuntimeError('Quantization algorithm from the OpenVINO backend '
+                           'does not support regular expressions in the ignored '
+                           'scopes yet')
+    if ignored_scope.types is not None:
+        ignored['operations'] = [{'type': type} for type in ignored_scope.types]
+    return ignored
+
+
+def _create_statistics_collector_config(
+    statistics_collector_params: StatisticsCollectorParameters) -> Dict[str, Any]:
+    """
+    Creates a statistic collector configuration.
+
+    :param statistics_collector_params: Statistic collector parameters
+    :return: A POT statistic collector configuration as dict.
+    """
+    config = {}
+    if statistics_collector_params.statistics_type is not None:
+        config['type'] = statistics_collector_params.statistics_type.value
+    if statistics_collector_params.aggregator_type is not None:
+        config['aggregator'] = statistics_collector_params.aggregator_type.value
+    if statistics_collector_params.clipping_value is not None:
+        config['clipping_value'] = statistics_collector_params.clipping_value
+    if statistics_collector_params.statistics_type in [
+            StatisticsType.QUANTILE, StatisticsType.ABS_QUANTILE]:
+        config['outlier_prob'] = statistics_collector_params.quantile_outlier_prob
```

Comment:
@alexsu52, we will use this `statistic_collecto_params` for `ov_native` backend, right?

Reply:
Yes, I suggest using POT notation. Do you have any concern?

Reply:
No concerns, looks doable 👍